src/widgets/concept-widget.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  VDomModel, VDomRenderer
+} from '@jupyterlab/apputils';
+import {INotebookTracker} from '@jupyterlab/notebook';
+
+import * as React from 'react';
+
+import {AllOfUsConfig} from '../config';
+import {Concept} from '../generated';
+import {ConceptsService} from '../services/concepts.service';
+
+import {Observable} from 'rxjs/Rx';
+
+export class ConceptsWidgetModel extends VDomModel {
+
+  public initialized: boolean;
+  public conceptsLoaded: boolean;
```

Comment:
conceptsLoaded and conceptsLoading seem like they should be mutually exclusive and therefore encoding redundant information. Do you need both? If so, maybe rename or doc